src/http.rs

```diff
@@ -394,3 +490,74 @@ mod http {
         assert_eq!(edn_body, expected);
     }
 }
+
+#[cfg(test)]
+mod build_url {
+    use super::build_timed_url;
+    use chrono::prelude::*;
+
+    #[test]
+    fn both_times_are_none() {
+        let url = build_timed_url("localhost:3000".to_string(), "entity", None, None);
+
+        assert_eq!(url, "localhost:3000/entity");
+    }
+
+    #[test]
+    fn both_times_are_some() {
+        let url = build_timed_url(
+            "localhost:3000".to_string(),
+            "entity",
+            Some(
+                "2020-08-09T18:05:29.301-03:00"
+                    .parse::<DateTime<FixedOffset>>()
+                    .unwrap(),
+            ),
+            Some(
+                "2020-08-09T18:05:29.301-03:00"
```

Comment:
What about sending two different types so the test becomes richer?

Reply:
Will evolve tests in another PR with examples. Noted

Reply:
Sorry, I've explained it poorly, I meant two different dates.

Reply:
Ah ok